### PR TITLE
update qovery guide

### PR DESCRIPTION
**Pull request by maryamsulemani97**
Copies @arnaudjnn's [PR](https://github.com/meilisearch/documentation/pull/1036) on updating the Qovery guide
 The original PR was closed as the branch seems to have been deleted by GitHub.
 
 Thank you @arnaudjnn! 😄

### CODE REVIEW

File: learn/cookbooks/qovery.md [hunk 1]
```diff
@@ -1,38 +1,64 @@
 # Deploy a Meilisearch instance on Qovery
 
-_The following is a guest post by Romaric Philogène, CEO and Co-Founder of Qovery._
+[Qovery](https://www.qovery.com) is a fully-managed cloud platform that runs on your AWS, Digital Ocean and Scaleway account where you can host static sites, backend APIs, databases, cron jobs, and all your other apps in one place.
```
Comment (dichotommy) [inline learn/cookbooks/qovery.md:3]:
```suggestion
[Qovery](https://www.qovery.com) is a fully-managed cloud platform that runs on your AWS, DigitalOcean, or Scaleway account where you can host static sites, backend APIs, databases, cron jobs, and all your other apps in one place.
```

File: learn/cookbooks/qovery.md [hunk 2]
```diff
@@ -1,38 +1,64 @@
...
 
-Visit [the Qovery dashboard](https://start.qovery.com) to create an account if you don't already have one.
+!["Create Project" button](https://hub.qovery.com/img/heroku/heroku-2.png)
 
-### 2. Create a project
+### 3. Create a new environment
+
+* Click on **Create environment** and give it a name (e.g. staging, production).
+
+!["Create environment" button](https://hub.qovery.com/img/heroku/heroku-3.png)
```
Comment (dichotommy) [inline learn/cookbooks/qovery.md:30]: As the current guardian of alt text would you make any changes to this @maryamsulemani97 ?

**Review by dichotommy — CHANGES_REQUESTED** [review]

File: learn/cookbooks/qovery.md (hunk 2 above)
Comment (maryamsulemani97) [reply learn/cookbooks/qovery.md:30]: No, I followed this format for other similar images. Ik it's not a good practice for the alt text to repeat what your content says, but until we think of a better alternative(maybe reword the actual text), I'd like to stay consistent

**Review by dichotommy — APPROVED** [review]
🧑‍🚀 🪐

**Comment by dichotommy** [comment]
bors merge

**Comment by dichotommy** [comment]
bors merge
